src/main/java/sirius/pasta/tagliatelle/emitter/LoopEmitter.java

```diff
@@ -91,6 +97,20 @@ protected void emitToContext(LocalRenderContext context) throws Exception {
         }
     }
 
+    private Iterable<?> getItems(Object innerExpression) {
+        if (innerExpression instanceof Iterable<?> iterable) {
+            return iterable;
+        }
+        if (innerExpression.getClass().isArray()) {
```

Comment:
Hier ist kein `getGenericType` nötig?

Reply:
innerExpression != iterableExpression :P
No, this works as is